### ISSUE TITLE
Claim win feature is needed if opponent player disconnects

### ISSUE THREAD

**Issue by gbtami** [gbtami/pychess-variants#575]

**Comment by Diwaditya** [comment]
@gbtami 
No! You shouldn't implement it! First, add a **“RECONNECTING”** when users disconnect, the blinking clock is barely noticeable.

**Comment by gbtami** [comment]
This issue is about claim win. For other feature requests open another issue please.
